packages/mesa/build.sh

```diff
@@ -29,12 +30,16 @@ TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
 -Dllvm=enabled
 -Dshared-llvm=enabled
 -Dplatforms=x11,wayland
--Dgallium-drivers=swrast,virgl,zink
--Dosmesa=true
+-Dgallium-drivers=llvmpipe,virgl,zink
```

Comment:
IIRC `swrast` is the same as `llvmpipe,softpipe`.

Reply:
Yes, but the `swrast` option itself is deprecated.

Reply:
IMO this line should be `-Dgallium-drivers=llvmpipe,softpipe,virgl,zink`